### PR TITLE
Adds SQS snippet for SQSBatchResponse

### PR DESCRIPTION
**Pull request by skamatka**
Issue #, if available:

Description of changes:
Added SQS Snipnet for SQS Batch Response

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: lambda-function-sqs-report-batch-item-failures/example.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+//composer install
+
+
+<?php
+
+use Bref\Context\Context;
+use Bref\Event\SqsEvent;
+
+
+function lambda_handler(SQSEvent $event, Context $context) {
```
Comment (falnyr) [inline lambda-function-sqs-report-batch-item-failures/example.php:12]: Based on the [Bref documentation](https://bref.sh/docs/runtimes/function#php-classes) You need to either return a function or a class for the handler. I suggest rewriting this handler into a class as it's more aligned to the other code samples for PHP.

File: lambda-function-sqs-report-batch-item-failures/example.php [hunk 2]
```diff
@@ -0,0 +1,29 @@
+//Make sure you have Bref installed and the necessary dependencies. Run the following commands in your project directory:
+//composer require bref/bref
+//composer install
+
+
+<?php
+
+use Bref\Context\Context;
+use Bref\Event\SqsEvent;
+
```
Comment (falnyr) [inline lambda-function-sqs-report-batch-item-failures/example.php:10]: You are missing the vendor autoloader to load these classes.

**Review by falnyr — CHANGES_REQUESTED** [review]
Hi @skamatka, thanks for your contribution. There are a few issues in the code that need addressing before we can merge this. Please check the subsequent comments for details. Feel free to reach out if you have any questions.

**Comment by skamatka** [comment]
I updated the code and tested it in my Isengard account. Please review the latest commit.

**Review by falnyr — APPROVED** [review]
Looks good to me, thanks for your contribution! 🙌

**Pull request merged**